pipelines/broad/arrays/single_sample/Arrays.documentation.md

```diff
@@ -26,52 +33,108 @@ The Arrays Pipeline takes the inputs described below. Inputs are grouped by func
+
+        *   regulatory_designation [Optional]. The regulatory designation, as specified by the Mercury LIMS.
+
+        *   research_project_id [Optional]. The research project ID, as specified by the Mercury LIMS.
     
 *   **Reference**
     *   ref_fasta. The cloud path of the reference FASTA used. Note: The pipeline is only designed to work with HG19 and has only been tested with it.
+
     *   ref_fasta_index. The cloud path of the index file for the reference FASTA.
+
     *   ref_fasta_dict. The cloud path of the dictionary file for the reference FASTA.
+
     *   dbSNP_vcf. The cloud path of the dbSnp VCF. Used for metrics collection.
+
     *   dbSNP_vcf_index. The cloud path of the index file for the dbSnp VCF.
 
 *   **Chip Metadata**
-    *   bead_pool_manifest_file. The cloud path of the Illumina bead pool manifest file for the genotyping array to be processed. 
-    *   extended_chip_manifest_file. The cloud path of the Illumina chip manifest for the genotyping array to be processed, in CSV form, as extended by the Picard tool CreateExtendedIlluminaManifest.
-    *   cluster_file. The cloud path of the Illumina cluster file (.egt) for the genotyping array to be processed.
-    *   gender_cluster_file [Optional]. The cloud path of the Illumina gender cluster file (.egt) for the genotyping array to be processed. This optional cluster file is used to run IAAP/gencall specifically for calling the gender of the sample.  It typically is designed with all assays other than those at specific sites on the sex chromosomes zeroed out.
-    *   zcall_thresholds_file [Optional]. The cloud path of a thresholds file to be used with zCall. [zCall](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463112/) is a rare variants caller that can be used to improve the calls on rare variant assays on the genotyping array. If this optional parameter is provided, the pipeline will run zCall using the specified thresholds file.
+    *   arrays_metadata_path [Optional]. The cloud path where chip-specific metadata will be looked for by the workflow under certain conditions as explained below:
+
+    *   bead_pool_manifest. The cloud path of the Illumina bead pool manifest file (.bpm) that will be used to process the array data is determined based on which of the following inputs are provided:
+
+        *   bead_pool_manifest_filename [Optional]. The **name** of the Illumina bead pool manifest file that will be used to process the array data. If specified, a file by this name will be looked for in arrays_metadata_path. If **NOT** specified, the pipeline will use the **bead_pool_manifest_file**.
+
+        *   bead_pool_manifest_file [Optional]. The cloud path of the Illumina bead pool manifest file for the genotyping array to be processed.
+
+    *   cluster. The cloud path of the Illumina cluster file (.egt) that will be used to process the array is determined based on which of the following inputs are provided:
```

Comment:
Is this still an input to the pipeline? I am only seeing `cluster_filename` and `cluster_file`?

Reply:
okay im now seeing that cluster, gender_cluster and zcall_threshold all have this where the optional inputs are listed below. potentially ignore all of these comments, assuming the pattern is on purpose!

Reply:
Yeah, I was trying to use a generic naming pattern for these cases where there are now two optional inputs.

Reply:
I updated the formatting to hopefully make the inputs clearer.